src/mongocxx/options/encrypt.cpp

```diff
@@ -51,24 +49,35 @@ const stdx::optional<encrypt::encryption_algorithm>& encrypt::algorithm() const
     return _algorithm;
 }
 
+const stdx::optional<bsoncxx::types::bson_value::view_or_value>& encrypt::key_id() const {
+    return _key_id;
+}
```

Comment:
Awesome! This resolved https://jira.mongodb.org/browse/CXX-2059 as well. Feel free to self-assign and close when this is merged.

Reply:
Oh right, I forgot we made a ticket for that, that's good.